projects/medical/2d_image/microscopy_images/bactteria_detection/tools/prepare_dataset.py

```diff
@@ -0,0 +1,33 @@
+import glob
+import os
+import shutil
+
+from PIL import Image
+
+root_path = 'data/'
+img_suffix = '.png'
+seg_map_suffix = '.png'
+save_img_suffix = '.png'
+save_seg_map_suffix = '.png'
+
+x_train = glob.glob(
+    'data/Bacteria_detection_with_darkfield_microscopy_datasets/images/*' +
```

Comment:
```suggestion
    'data/images/*' +
```
am i right?

Reply:
‘data/Bacteria_detection_with_darkfield_microscopy_datasets/images/’ is right.

Reply:
please tell other users the path after dataset unpack is data/Bacteria_detection_with_darkfield_microscopy_datasets/